cli/src/commands.rs

```diff
@@ -0,0 +1,371 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+extern crate chrono;
+use crate::{command_utils::*, trusted_commands, trusted_commands::TrustedArgs, Cli};
+use base58::{FromBase58, ToBase58};
+use chrono::{DateTime, Utc};
+use clap::Subcommand;
+use codec::{Decode, Encode};
+use ita_stf::ShardIdentifier;
+use itc_rpc_client::direct_client::DirectApi;
+use itp_node_api_extensions::{PalletTeerexApi, TEEREX};
+use log::*;
+use my_node_runtime::{Balance, BalancesCall, Call, Event, Hash};
+use sp_application_crypto::{ed25519, sr25519};
+use sp_core::{crypto::Ss58Codec, sr25519 as sr25519_core, Pair, H256};
+use sp_keyring::AccountKeyring;
+use std::{
+	path::PathBuf,
+	sync::mpsc::channel,
+	time::{Duration, UNIX_EPOCH},
+};
+use substrate_api_client::{
+	compose_extrinsic, compose_extrinsic_offline, utils::FromHexString, GenericAddress, Metadata,
+	UncheckedExtrinsicV4, XtStatus,
+};
+use substrate_client_keystore::{KeystoreExt, LocalKeystore};
+
+const PREFUNDING_AMOUNT: u128 = 1_000_000_000;
+
+#[derive(Subcommand)]
+pub enum Commands {
+	/// query on-chain balance for AccountId
+	Balance {
+		/// AccountId in ss58check format
+		account: String,
+	},
+
+	/// generates a new account for the integritee chain
```

Comment:
I think this is too ambiguous. It should be made clear that this account is local in the keystore. There is no parentchain interaction.